apps/api/app/services/gaia_knowledge_service.py

```
@@ -0,0 +1,157 @@
+
+            # Extract texts and metadatas
+            texts = [item["content"] for item in items]
+            metadatas = [item.get("metadata", {}) for item in items]
+
+            # Add documents in batch
+            await client.aadd_texts(texts=texts, metadatas=metadatas)
+
+            logger.info(f"Added {len(items)} knowledge items to ChromaDB")
+            return len(items)
+
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)